# Comments provide naming from Technical Customer Information

# mdThetaIndcrMIMO
float32 theta_indicator_mimo

# mdThetaIndcrMIMOVldFlg
bool theta_indicator_mimo_valid

# mdPhiIndcr
float32 phi_indicator

# mdPhiIndcrVldFlg
bool phi_indicator_valid

# nRefIndcr
float32 reflections_indicator

# nRefIndcrVldFlg
bool reflections_indicator_valid

# thetaMIMORate
float32 theta_mimo_rate

# thetaMIMORteVldFlag
bool theta_mimo_rate_valid
